docs/guide/local/session-data.md

```diff
@@ -36,11 +38,13 @@ export default defineNuxtConfig({
   auth: {
     provider: {
       type: 'local',
-      sessionDataType: {
-        id: 'string | number',
-        firstName: 'string',
-        lastName: 'string',
-        subscriptions: '{ id: number, active: boolean}[]'
+      session: {
+        data: {
```

Comment:
```suggestion
        dataType: {
```

Same here 😊

Reply:
At lest I was consistently wrong with it 😂